example/ck_tile/03_gemm/universal_gemm.cpp

```diff
@@ -27,12 +27,16 @@ void try_run(ck_tile::TailNumber tn)
 
 template <typename ADataType,
           typename BDataType,
+          typename DsDataType,
           typename AccDataType,
           typename CDataType,
           typename ALayout,
           typename BLayout,
-          typename CLayout>
-float gemm_calc(const ck_tile::GemmHostArgs& args, const ck_tile::stream_config& s)
+          typename DsLayout,
+          typename CLayout,
```

Comment:
Please change accordingly throughout the PR. 
```suggestion
          typename ELayout,
```